src/BG/NRARFCNs.ts

```diff
@@ -3,8 +3,7 @@ import { mapArfcnListToBand } from "../utils/mapArfcnListToBand";
 
 const n3: SimpleArfcnDataItem[] = [
   {
-    arfcn: 368410,
-    bandwidth: 15,
+    bandwidth: 20,
     operator: "Vivacom BG",
     description: "DSS n3 in urban areas and on highways",
   },
```

Comment:
For the package to build, we either need to remove this entry, or add an NRARFCN to this entry.